src/Microsoft.Health.Fhir.Api/Resources.resx

```diff
@@ -179,6 +182,9 @@
   <data name="ToggleNavigation" xml:space="preserve">
     <value>Toggle navigation</value>
   </data>
+  <data name="TooManyCustomAuditHeaders" xml:space="preserve">
+    <value>A maximum of {0} custom audit headers allowed in the request.  {1} headers were detected.  Please retry the request with fewer custom headers.</value>
+  </data>
```

Comment:
Leave some comment about what {0} and {1} would be for the translators. An example would be nice.

Reply:
Agreed - Explaining what the placeholders are allows translators to have context when translating. Remember to not actually use curly braces as part of the explanation in the Comments field as they would then be interpreted as rules to be parsed - Instead refer to placeholders as (say) PH0, PH1. Also add the following rule to ensure that the placeholders remain in the translation: {NumberedPlaceHolder=”{0}","{1}”} Ref: aka.ms/NExTLoc

Reply:
This convention for explaining and controlling placeholders follows throughout this file @feordin